distribution/docs/src/main/resources/content/_configuring/configuring-idp.adoc

```diff
@@ -69,37 +68,37 @@ Install the included IdP implementation.
 </md:EntityDescriptor>
 ----
 
-.Configuring IdP Authentication Types
-Set the authentication types that will be accepted by the IdP.
+.Configuring IdP as the Authentication Type
+To use the IdP for authentication,
 
-. Navigate to the Admin Console.
-. Select the Security application.
-. Select the Configuration tab.
-. Select Web Context Policy Manager
-. Under Authentication Types, set the IDP authentication type as necessary. Note that it should only be used on context paths that will be accessed by users via web browsers. For example:
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-security}* application.
+. Select the *Configuration* tab.
+. Select *Web Context Policy Manager*
+. Under *Authentication Types*, set the IDP authentication type to context paths as necessary.
+Note that it should only be used on context paths that will be accessed by users via web browsers. For example:
 * `/search=IDP`
 
 Other authentication types can also be used in conjunction with the IDP type.
 For example, if you wanted to secure the entire system with the IDP, but still allow legacy clients that don’t understand the SAML ECP specification to connect, you could set `/=IDP|PKI`.
-With that configuration, any clients that failed to connect using either the SAML 2.0 Web SSO Profile or the SAML ECP spec would fall back to 2-way TLS for authentication.
+With that configuration, any clients that failed to connect using either the SAML 2.0 Web SSO Profile or the SAML ECP specification would fall back to 2-way TLS for authentication.
 
 [NOTE]
 ====
-If you have configured /search to use IDP, ensure to select the "External Authentication" checkbox in Search UI standard settings.
+If you have configured `/search` to use IDP, ensure to select the "External Authentication" checkbox in Search UI standard settings.
 ====
 
-.[[_configuring_idp_sp]]Configuring Security IdP Service Provider (SP)
-The IdP client that interacts with the specified Identity Provider.
-
-. Navigate to Admin Console.
-. Select the Security application.
-. Select the Configuration tab.
-. Select IdP Client.
-. Populate IdP Metadata field through one of the following:
-.. An HTTPS URL (https://)
-.. A file URL (file:)
-.. An XML block to refer to desired metadata, such as:
-... e.g., https://localhost:8993/services/idp/login/metadata
+.[[_configuring_idp_sp]]Configuring the Service Providers (SP)
+To configure the IdP client that interacts with the specified Identity Provider also known as the Service Provider,
```

Comment:
✏️ "To configure the IdP client **(also known as the Service Provider)** that interacts with the specified Identity Provider,"